src/main/webapp/help/ArtifactoryBuilder/help-pipelinesCbkUrl.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Specify the callback URL of your JFrog Pipelines installation, like <tt>http://jfrog-platform-url/pipelines/api/v1/projectIntegrations/2957/hook</tt>
```

Comment:
...the callback URL... --> ...the URL...

Let's also describe here how to get this URL from JFrog Pipeline's UI.

Reply:
Changed to `integration URL`